src/content/docs/service-level-management/create-slm.mdx

```diff
@@ -423,7 +427,7 @@ Follow these steps:
 
 ### Edit SLIs [#edit-sli]
 
-Once you’ve created an SLI you can edit it either through the service levels list page, by clicking on the **...** menu and then `Edit`,
+Once you've created an SLI you can edit it either through the service levels list page, by clicking on the **...** menu and then `Edit`,
```

Comment:
@zuluecho9 I got confused by the sentence that seemed to end without a complete thought followed by an image. How about just making a bullet list of the two ways and indenting the images under each bullet:

* Click on the ... menu, and then Edit
* Go to the summary page and click Edit